turbopack/crates/turbopack-core/src/ident.rs

```diff
@@ -103,7 +103,7 @@ impl ValueToString for AssetIdent {
                 let part = part.to_string().await?;
                 // facade is not included in ident as switching between facade and non-facade
                 // shouldn't change the ident
-                if part.as_str() != "facade" {
+                if part != "facade" {
```

Comment:
That check could be a `matches!(..., ModulePart::Facade)` and skip the to_string in that case